Docs/docs/worlds/udon/vm-and-assembly/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "The Udon VM and Udon Assembly"
+sidebar_position: 1
+createdAt: "2024-03-18T10:00:00.000Z"
+updatedAt: "2024-03-18T10:00:00.000Z"
+---
+## Overview of the Udon VM
```

Comment:
```suggestion

:::info

This page was written by [20kdc](https://github.com/20kdc), a member of the VRChat community. Thank you for your contribution!

The VRChat team can't guarantee the validity of the information on this page. If you would like to suggest changes, go to the bottom of this page and click "Edit this page."
:::

## Overview of the Udon VM
```

Reply:
Is there any reason to specifically namedrop myself? Future contributors may appear at some point. (In the meantime, I'll commit a slight modification of this.)

Reply:
That's fine! You're right, there's a chance that someone else might edit the page.